main.go

```diff
@@ -175,13 +209,23 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-
 	// create the faucet instance
-	f := faucet{
-		signer:     &signer,
-		authTypes:  authTypes,
-		waitPeriod: waitPeriod,
-		storage:    storage,
+	f := faucet.Faucet{
+		Signer:     &signer,
+		AuthTypes:  authTypes,
+		WaitPeriod: waitPeriod,
+		Storage:    storage,
+	}
+	var s *stripehandler.StripeHandler
+	if f.AuthTypes[faucet.AuthTypeStripe] > 0 {
+		s, err = stripehandler.NewStripeClient(stripeKey, stripePriceId, stripeWebhookSecret, stripeMinQuantity, stripeMaxQuantity, stripeDefaultQuantity, &f, storage)
+		if err != nil {
+			log.Warnf("stripe initialization error: %s", err)
+		} else {
+			log.Infof("stripe enabled with price id %s", stripePriceId)
+		}
+	} else {
+		log.Info("stripe not configured")
```

Comment:
Do not print this, please. Either you print all handlers that have not been configured or none.